tests/test_olsr.py

```diff
@@ -18,8 +22,16 @@
 ).read()
 
 
+def parameterized_test_name_func(testcase_func, param_num, param):
+    format = 'newformat' if 'version' in param[0][0] else 'oldformat'
+    return f'{testcase_func.__name__}_{param_num}_{format}'
```

Comment:
I have added this to make the name of test case readable in verbose mode. Without this, the whole content of `param` was being appended to the test case name in verbose mode.